docs/harvesting.md

```diff
@@ -150,7 +150,7 @@ Your method for doing this will depend on the shape and size of the data you're
 
 If you encounter a page that links to lots of data (for example a "downloads" page), this approach may work well. It's important to only use this approach when you encounter *data*, for example PDF's, .zip archives, .csv datasets, etc.
 
-The tricky part of this approach is generating a list of URLs to download from the page. If you're skilled with using scripts in combination with html-parsers (for example python's wonderful [beautiful-soup package](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#quick-start)), go for it. Otherwise, we've included the [jquery-URL-extraction guide](thttps://github.com/edgi-govdata-archiving/harvesting-tools/tree/master/jquery-URL-extraction)], which has the advantage of working within a browser and can operate on a page that has been modified by JavaScript.
+The tricky part of this approach is generating a list of URLs to download from the page. If you're skilled with using scripts in combination with html-parsers (for example python's wonderful [beautiful-soup package](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#quick-start)), go for it. Otherwise, we've included the [jquery-URL-extraction guide](thttps://github.com/edgi-govdata-archiving/harvesting-tools/tree/master/jquery-URL-extraction), which has the advantage of working within a browser and can operate on a page that has been modified by JavaScript.
```

Comment:
Looks like that jQuery URL needs to be updated from:
thttps://github.com/edgi-govdata-archiving/harvesting-tools/tree/master/jquery-URL-extraction
to:
https://github.com/edgi-govdata-archiving/harvesting-tools/tree/master/jquery-url-extraction

(and below for case-sensitivity)